Android/res/values-es/strings.xml

```diff
@@ -0,0 +1,454 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_live_upload">Habilitar la subida en vivo (ALPHA TEST)</string>
+    <string name="Pause">Pausa</string>
+    <string name="about">Sobre</string>
+    <string name="account_label">Cuenta</string>
+    <string name="activity_title_map_provider_preferences">Preferencias del Proveedor de Mapas</string>
+    <string name="air_speed_label">Velocidad Aire</string>
```

Comment:
I think the right term for this is "velocidad aerodinámica", but I might be wrong.